workflows/SUEP_coffea_ZH_simple.py

```diff
@@ -821,11 +829,39 @@ def doTracksDropping(self, events, tracks):
         passHighPtTracks = highPtTracks[(randomHigh > probsHighPt[self.era])]
         # And then broadcast them together again
         return {"":tracks, "_TRACKUP":ak.concatenate([passLowPtTracks, passHighPtTracks], axis=1)}
-        
+    
+    def doTriggerSFs(self, electrons, muons):
+        ceval = correctionlib.CorrectionSet.from_file("/eos/user/g/gdecastr/SUEPCoffea_dask/triggerEfficiencyStudy/masterJSON.json") ## Need to fill in path
```

Comment:
Don't use absolute paths that depend on your user space (i.e. this thing is not usable for anybody but you because you are using an absolute path to a file on your local storage). Something like correctionlib.CorrectionSet.from_file("triggerEfficiencyStudy/masterJSON.json") is more flexible